clang/lib/Driver/ToolChains/CommonArgs.cpp

```diff
@@ -1029,7 +1042,7 @@ void tools::addFortranRuntimeLibraryPath(const ToolChain &TC,
                                          ArgStringList &CmdArgs) {
   // Default to the <driver-path>/../lib directory. This works fine on the
   // platforms that we have tested so far. We will probably have to re-fine
-  // this in the future. In particular, on some platforms, we may need to use
+  // this in the future. In particular, on some platforms, we may need to useq
```

Comment:
Nit: accidental change?

Reply:
Yes, indeed.  Sorry about that!

I have also found typos in my comment.  Fixed these, too.